src/PrestaShopBundle/Resources/config/services/core/constraint_validator.yml

```diff
@@ -58,3 +58,11 @@ services:
       - '@prestashop.adapter.country.country_required_fields_provider'
     tags:
       - { name: validator.constraint_validator }
+
+  prestashop.core.constraint_validator.clean_html_validator:
+      class: 'PrestaShop\PrestaShop\Core\ConstraintValidator\CleanHtmlValidator'
+      arguments:
+        - "@=service('prestashop.adapter.legacy.configuration').getBoolean('PS_ALLOW_HTML_IFRAME')"
```

Comment:
It's a BC break :)
As to be documented

Reply:
Updated PR's description, do I need to document it somewhere else?